### ISSUE TITLE
accessibility for visual impairment

### ISSUE THREAD

**Issue by IDADV** [microsoft/vscode#237487]
Hello, I am introducing myself, my name is Youssef and I am visually impaired. I am contacting you because I have an idea to propose to you. I would like us to work together to adapt your software for the visually impaired. Why am I sending you this message? I am sending you this message because for us visually impaired people it is not easy to use unsuitable software and we cannot sometimes do what we like because the software is not accessible to us and I would like to adapt and make this accessibility that is not there. I hope you will accept that we work together. I am waiting for your response. Thank you and have a nice day.

Best regards
Youssef

**Comment by meganrogge** [comment]
Hi Youssef, if you have specific suggestions, it would be best if you create detailed issues for each on this repository. I work hand in hand with many visually impaired developers and learn a lot to improve our product.
